models/articles-model.js

```diff
@@ -18,6 +19,32 @@ function retrieveArticles(article_id) {
     })
 }
 
+function newComment(article_id, article_body) {
+    
+    const criteria = ['author', 'body']
+    if (!criteria.every(objCriteria => Object.keys(article_body).includes(objCriteria))) {
+        return Promise.reject({status: 400, msg: "Bad Request"})
+    }
+    
+    const formattedComment = [
+        article_body.body,
+        article_id,
+        article_body.author,
+        0,
+        new Date().toISOString()
+    ]
+
+    const queryString = format(
+        `INSERT INTO comments
+        (body, article_id, author, votes, created_at)
```

Comment:
`votes` and `created_at` will both be inserted using default values, so you don't need to include them in this query, PSQL has got you covered 😎